_posts/2024-04-19-keyed-servicebehavior.markdown

```diff
@@ -0,0 +1,41 @@
+---
+# Layout
+layout: post
+title:  "Add 'Keyed by Service type' ServiceBehavior support in CoreWCF"
+date:   2024-04-19 13:00:00 +0200
```

Comment:
Update the date to something more current

Reply:
Updated!